src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient.swift

```diff
@@ -406,6 +403,25 @@ final class HTTPClient: Client {
 
 // MARK: Extensions
 
+private extension HTTPClient {
+	enum Model {
```

Comment:
I would remove this enum. It brings nothing.

Reply:
To be more clear. :) Just the enum, the structs inside are fine, but only outside of the enum. I would also consider bringing them out of ```HTTPClient```, but that could stay as it is.

Reply:
okay, I wanted to use the Model enum as a "namespace" for the different response structs but I will remove it when you do not see the necessity!

Reply:
It's too much nesting like that, and in reality it's not a model, since you don't pass it anywhere, but more like an API entity.